src/tests/functional/plugin/shared/src/behavior/ov_infer_request/memory_states.cpp

```diff
@@ -0,0 +1,292 @@
+// // Copyright (C) 2018-2023 Intel Corporation
+// // SPDX-License-Identifier: Apache-2.0
+// //
+
+#include "behavior/ov_infer_request/memory_states.hpp"
+
+#include <base/behavior_test_utils.hpp>
```

Comment:
```suggestion
#include "base/behavior_test_utils.hpp"
```
Detail: use `""` for OV includes